src/components/util/search-matching-equipments.js

```diff
@@ -26,40 +27,42 @@ export const useSearchMatchingEquipments = (
     const [useNameLocal] = useParameterState(PARAM_USE_NAME);
 
     const searchMatchingEquipments = useCallback(
-        (searchTerm) => {
+        (searchTerm, sooner = false) => {
             clearTimeout(timer.current);
 
-            timer.current = setTimeout(() => {
-                lastSearchTermRef.current = searchTerm;
-                fetchEquipmentsInfos(
-                    studyUuid,
-                    nodeUuid,
-                    searchTerm,
-                    useNameLocal,
-                    inUpstreamBuiltParentNode,
-                    equipmentType
-                )
-                    .then((infos) => {
-                        if (searchTerm === lastSearchTermRef.current) {
-                            setEquipmentsFound(
-                                getEquipmentsInfosForSearchBar(
-                                    infos,
-                                    useNameLocal
-                                )
-                            );
-                        } // else ignore results of outdated fetch
-                    })
-                    .catch((errorMessage) =>
-                        snackError(errorMessage, 'equipmentsSearchingError')
-                    );
-            }, SEARCH_FETCH_TIMEOUT);
+            timer.current = setTimeout(
+                () => {
+                    lastSearchTermRef.current = searchTerm;
+                    fetchEquipmentsInfos(
+                        studyUuid,
+                        nodeUuid,
+                        searchTerm,
+                        useNameLocal,
+                        inUpstreamBuiltParentNode,
+                        equipmentType
+                    )
+                        .then((infos) => {
+                            if (searchTerm === lastSearchTermRef.current) {
+                                setEquipmentsFound(
+                                    makeItems(infos, useNameLocal)
+                                );
+                            } // else ignore results of outdated fetch
+                            timer.current = undefined;
```

Comment:
why ? What is the difference with "clearTimeout(timer.current)" ?

Reply:
Uh, it was just for printf-debugging, it allowed to know if a request was on air.
I am removing it.
For the difference with `clearTimeout(timer.current)`, I guess there is more than one, actually